src/state_channels/pp_sc_packet_handler.erl

```diff
@@ -50,32 +50,37 @@ handle_packet(SCPacket, PacketTime, Pid) ->
     case pp_config:lookup(RoutingInfo) of
         {error, {buying_inactive, NetID}} ->
             lager:debug(
+                [{packet_type, PacketType}, {net_id, NetID}, {error, buying_inactive}],
```

Comment:
You could stuff all that in `lager:md`

Reply:
That would attach it to the state channel process calling our handler.
I only want it for these long lines.